src/main/resources/apps/pantheon/modules/all/html.html

```diff
@@ -10,35 +10,89 @@ <h1>All Modules</h1>
         <a class="btn btn-primary" href="/modules.add.html">New Module</a>
     </p>
 
-    <!-- <div class="list-group list-view-pf list-view-pf-view" data-sly-list="${resource.listChildren}"> -->
-    <div class="list-group list-view-pf list-view-pf-view" data-sly-list="${moduleData.getModulesCreateSort}">
-        <div class="list-group-item">
-            <a href="${item.name @ prependPath='modules/', addSelectors=['preview']}">
-                <div class="list-view-pf-actions">
-                    <div class="list-view-pf-main-info">
-                        <div class="list-view-pf-left">
-                            <!--span class="fa fa-plane list-view-pf-icon-sm"></span-->
-                        </div>
-                        <div class="list-view-pf-body">
-                            <div class="list-view-pf-description">
-                                <div class="list-group-item-heading">
-                                    ${item.jcr:title}
-                                </div>
-                            </div>
-                            <div class="list-view-pf-additional-info">
-                                <div class="list-group-item-text">
-                                    ${item.jcr:description}
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </a>
-        </div>
-    </div>
+    <table id="myTable">
+        <tr>
+            <th width="200px"><a href="#" onclick="sortTable(0)">Name</a></th>
+            <th width="325px"><a href="#" onclick="sortTable(1)">Description</th>
+            <th><a href="#" onclick="sortTable(2)">Upload Time</a></th>
+        </tr>
+
+        <sly data-sly-list="${moduleData.getModulesCreateSort}">
+            <tr>
+                <td>
+                    <a href="${item.name @ prependPath='modules/', addSelectors=['preview']}">${item.jcr:title}</a>
+                </td>
+                <td>
+                    <a href="${item.name @ prependPath='modules/', addSelectors=['preview']}">${item.jcr:description}</a>
+                </td>
+                <td>
+                    <sly data-sly-use.date="${'com.redhat.pantheon.use.DateFormatter' @ format='yyyy/MM/dd HH:mm:ss', date=item.jcr:created}">${date.value}</sly>
+                </td>
+            </tr>
+        </sly>
+    </table>
 </div>
 
 <div data-sly-include="${'/apps/pantheon/partials/footer.html'}">
 </div>
+
+<!-- https://www.w3schools.com/howto/howto_js_sort_table.asp -->
+<script>
+function sortTable(n) {
```

Comment:
This is the type of code we need to have unit tested and inlining it doesn't help with that. This is where a good front end framework helps (e.g. Angular, etc). We might need to discover how to do this while continuing to take advantage of HTL.

Reply:
Yeah, I knew this wasn't the greatest solution, but my thought was I'll get something quick and dirty in now, and we can implement a proper UI in a future story.